tests/test_io/test_spikegadgets.py

```diff
@@ -0,0 +1,35 @@
+from pathlib import Path
+from xml.etree import ElementTree
+
+import pytest
+
+from probeinterface import (
+    read_spikegadgets,
+    parse_spikegadgets_header,
```

Comment:
you can still access this for tests with `from probeinterface.io import parse_spikegadgets_header`

Reply:
You mean, even if it is removed from __init__.py?

Reply:
Yes, see the failing test